### DISCUSSION TITLE
Options type column

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#8167] [Help] [answered]
Hi everyone.
I have an options type column in a table called (equipment). I want to add an option to this column when a row has been created in the equipment table

**Comment by melohagan** [comment, marked as answer]
Hey @rafikApp 

If you want to have a 'dynamic' options list, then you'll need to use a *Data provider* as the options source: https://docs.budibase.com/docs/option-picker#using-a-data-provider

Then when you add a row to your equipment table it will add an option within your picker.
